src/attachment/index.stories.js

```diff
@@ -0,0 +1,59 @@
+import "./index.ts";
+import { html } from "lit-html";
+
+export default {
+  title: "desktop/attachment",
+  argTypes: {
+    className: { name: "className" },
+    error: { name: "error" },
+    id: { name: "id" },
+    label: { name: "label" },
+    language: {
+      name: "language",
+      control: {
+        type: "select",
+        options: ["auto", "en", "ja", "zh", "zh-TW"],
+      },
+    },
+    requiredIcon: { name: "requiredIcon" },
+    files: { name: "files" },
+    visible: { name: "visible" },
+    disabled: { name: "disabled" },
+  },
+  parameters: {
+    actions: {
+      handles: ["change"],
+    },
+  },
+};
+const Template = (args) => {
+  const handleAttachmentChange = (event) => {
+    console.log(event);
+  };
+  return html`
```

Comment:
change to new way storybook:const Template = (args) => {
  const attachment = new Attachment({ ...args });
  attachment.addEventListener("change", (event) => {
    console.log(event, "event");
  });
  return attachment;
};